src/NuGet.Clients/NuGet.CommandLine/NuGetCommand.resx

```diff
@@ -5245,4 +5245,18 @@ nuget update -Self</value>
   <data name="CommandMSBuildVersion" xml:space="preserve">
     <value>Specifies the version of MSBuild to be used with this command. Supported values are 4, 12, 14. By default the MSBuild in your path is picked, otherwise it defaults to the highest installed version of MSBuild.</value>
   </data>
+  <data name="ClearCacheCommandDescription" xml:space="preserve">
+    <value>Clears your local NuGet cache.</value>
```

Comment:
I would elaborate a bit more here and describe the machine cache and web cache that will be cleared briefly.